src/StreamJsonRpc/JsonMessageFormatter.cs

```diff
@@ -388,6 +407,31 @@ private JsonRpcRequest ReadRequest(JToken json)
                 args is JArray argsArray ? (object)PartiallyParsePositionalArguments(argsArray) :
                 null;
 
+            // If method is $/progress, get the progress instance from the dictionary and call Report
+            string method = json.Value<string>("method");
+
+            if (method.Equals("$/progress"))
```

Comment:
>progress [](start = 33, length = 8)

Can we expose JsonRpc.ProgressRequestSpecialMethod to be internal and use that here instead of hardcoding the value?

Reply:
This too has been addressed already in the latest update.

---
In reply to: [304567313](https://github.com/milopezc/vs-streamjsonrpc/pull/1#discussion_r304567313) [](ancestors = 304567313)